pkg/operator/osd/osd.go

```diff
@@ -97,6 +99,10 @@ func (c *Cluster) osdContainer(cluster *mon.ClusterInfo) v1.Container {
 
 	command := fmt.Sprintf("/usr/bin/rookd osd --data-dir=%s --mon-endpoints=%s --cluster-name=%s ",
 		k8sutil.DataDir, mon.FlattenMonEndpoints(cluster.Monitors), cluster.Name)
+	if c.useAllDevices {
+		command += fmt.Sprintf("--data-devices=all")
```

Comment:
should this be added with a trailing space in case other command args get added on in the future?